Added content to the Sponsorship/Prospectus Page

Hellrungj opened the pull request:
- Added Limited to Row
- Added A La Carte Sponsorship Options Table   
  - Added Limited to row
  - Added Branding in conference media
  - Added Branding on t-shirt
  - Added Conference ticket
#35 Update prospectus for 2020

Hellrungj committed:
Added the content to the Sponsorship/Prospectus Page
>> - Added Limited to Row
>> - Added A La Carte Sponsorship Options Table
>>   - Added Limited to row
>>   - Added Branding in conference media
>>   - Added Branding on t-shirt
>>   - Added Conference ticket

kwelch commented on src/pages/become-a-sponsor.js:
```diff
@@ -118,6 +118,29 @@ export default () => {
+                  &#8734;
+                </span>
+              </td>
+            </tr>
```
I think I would rather have them reach out or we cross of the row when sold out than to display the counts.

lapinell replied:
I'm going to remove this row then. And we can add a call to action elsewhere.

kwelch commented on src/pages/become-a-sponsor.js:
```diff
@@ -104,7 +104,7 @@ export default () => {
             <tr>
               <th>Benefit</th>
-              <th css={styles.highlight}>Platinum (Limit 1)</th>
+              <th css={styles.highlight}>Platinum</th>
```
I think leaving limit one here may be better than listing limits on all of them.

kwelch commented on src/pages/become-a-sponsor.js:
```diff
@@ -248,6 +271,150 @@ export default () => {
           </tbody>
         </table>
 
+        <Styled.h3>A La Carte Sponsorship Options</Styled.h3>
```
This section replaces the `Exclusive Sponsorship options`, so we should probably remove it: https://github.com/undergroundjs/website/pull/38/files#diff-c0a411ccae889ddca60b183f7439007fR90-R100

kwelch commented on src/pages/become-a-sponsor.js:
```diff
@@ -248,6 +271,150 @@ export default () => {
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
```
the aria-label is only needed to wrap the emoji so those using a screenreader know what the check means. We should remove it in the cases where it is wrapping a number because it will override the number and it will lose meaning.

kwelch reviewed (commented):
Thank you so much for working on this! I left some comments with some thoughts I had. Much appreciated.

Hellrungj committed:
Remove and Update Sponsorship/Prospectus page
Added Limit 1 to the Platinum Cal Heading
Removed A La Carte Sponsorship Option Table
Update the numbers in the table to not include image span

Hellrungj commented:
Updated the Sponsorship/Prospectus Page. 
[Commit Link](https://github.com/undergroundjs/website/pull/38/commits/c5e0a9817d24f396d35e2e9acab2cc3ef96cf49a)

kwelch committed:
Delete package-lock.json

kwelch commented:
I am good with this as is for now. Can we add a link to this in the footer.
